faunadb/query.py

```diff
@@ -40,6 +43,45 @@ def quote(expr):
   """See the `docs <https://faunadb.com/documentation/queries#basic_forms>`__."""
   return {"quote": expr}
 
+thread_local = local()
```

Comment:
I think we might as well move this into the function instead of having it on the module scope. If it stays in the module scope though I would like to see it at the top below the imports, before we start defining methods.